.github/workflows/build_apps.yml

```
@@ -0,0 +1,97 @@
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+          NEW_TAG="${{ steps.increment_tag.outputs.new_tag }}"
+
+          echo "Tag pushed to GitHub: $NEW_TAG"
+          git tag "$NEW_TAG"
+          git push origin "$NEW_TAG"
+
+  build-frontend:
+    name: "Build Docker Image"
+    if: ${{ github.repository == format('{0}/inbox-zero', env.DOCKER_USERNAME) }}
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)